src/script.js

```diff
@@ -1,98 +1,76 @@
+        var phoneNumber = document.querySelector('.input-phone').value;
+        var email = document.querySelector('.input-email').value;
+        var emotions = document.querySelector('.text-describe').value;
 
-    var impression = getValue();
-    var sights = getSights();
+        var impression = getValue();
+        var sights = getSights();
 
-    var body = 'name=' + name + '&surname=' + surname + '&patronym=' + patronym +
-        '&phoneNumber=' + phoneNumber + '&email=' + email + '&emotions=' +
-        emotions + '&impression=' + impression + '&sights' + sights;
+        var body = 'name=' + name + '&surname=' + surname + '&patronym=' + patronym +
+            '&phoneNumber=' + phoneNumber + '&email=' + email + '&emotions=' +
+            emotions + '&impression=' + impression + '&sights' + sights;
 
-    var xhr = new XMLHttpRequest();
+        var xhr = new XMLHttpRequest();
 
-    xhr.open('POST', '#');
-    xhr.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
+        xhr.open('POST', '#');
+        xhr.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
 
-    xhr.send(body);
+        xhr.send(body);
 
-    xhr.onload = function () {
-        if (xhr.status != 200) {
-            alert(`Error' + xhr.status + ':'+ xhr.statusText`);
-        } else {
-            alert(`Done`);
-        }
-    };
-
-    xhr.onerror = function () {
-        alert("Request failed");
-    };
-
-})
-
-var menuIcon = document.querySelector('.icon-menu');
-
-
-menuIcon.addEventListener('click', function(){
-    var nextSibling = document.querySelector('.wrapper');
-    var parentElem = document.body;
-    var mobileMenu = document.createElement('div');
-    mobileMenu.className = 'mobile-menu';
-    var mobileMenuItem1 = document.createElement('div');
-    mobileMenuItem1.className = 'blue-item';
-    mobileMenuItem1.innerHTML = "главная";
-    parentElem.insertBefore(mobileMenu, nextSibling);
-    mobileMenu.insertBefore(mobileMenuItem1, null);
-    var mobileMenuItem2 = mobileMenuItem1.cloneNode(true);
-    mobileMenuItem2.innerHTML = 'фото и видео';
-    mobileMenu.appendChild(mobileMenuItem2);
-    var mobileMenuItem3 = mobileMenuItem1.cloneNode(true);
-    mobileMenuItem3.innerHTML = 'форма отзыва';
-    mobileMenu.appendChild(mobileMenuItem3);
-    var mobileMenuItem4 = mobileMenuItem1.cloneNode(true);
-    mobileMenuItem4.innerHTML = 'html academy';
-    mobileMenu.appendChild(mobileMenuItem4);
-    var cross = document.createElement('img');
-    cross.className = 'cross';
-    cross.src = "public/img/icon-cross.png";
-    mobileMenuItem1.appendChild(cross);
-
-    cross.onclick = function() {
-    var mobileMenu = document.querySelector('.mobile-menu');
-    var parentElem = document.body;
-    parentElem.removeChild(mobileMenu);
-}
-
-})
+        xhr.onload = function () {
+            if (xhr.status != 200) {
+                alert(`Error' + xhr.status + ':'+ xhr.statusText`);
```

Comment:
please learn this chapter and use quotes correctly: http://jsraccoon.ru/es6-interpolation